src/project-css/app/Helpers/CollectionHelper.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * @author Ajay Krishna Teja Kavur
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Helpers;
+
+use Illuminate\Support\Facades\Storage;
+use App\Models\Collection;
+
+class CollectionHelper {
+    /**
+     * Collection Helper
+     *
+     * These are various functions that help with creating and modifying 
+     * collections.
+     *
+     */
+
+     public function create($data) {
+        // Create the collection name
+        $thisClctn = new Collection;
+        $thisClctn->clctnName = $data['name'];
+        $thisClctn->isCms = $data['isCms'];
+
+        // set cmsId if it exists
+        if (array_key_exists('cmsId', $data)) {
+          $thisClctn->cmsId = $data['cmsId'];
+        }
+
+        $thisClctn->save();
+
+        // create folder in storage that will contain any additional files associated to the collection
+        if (Storage::exists($thisClctn->clctnName) == FALSE) {
+          Storage::makeDirectory($thisClctn->clctnName, 0775);
+        }
+
+        return $thisClctn;
+     }
+
+     public function update($data) {
```

Comment:
Needs Documentation: 
Be clear on what the parameters of the function are as well as what the return values are supposed to be.